ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/button/v1/button/button.html

```diff
@@ -35,7 +35,7 @@
       title="${button.tooltipVisible ? '' :   button.tooltipText}"
       aria-label="${button.label.value}">
     <sly data-sly-call="${iconTemplate.icon @ icon=button.icon}"></sly>
-    <span data-sly-test="${button.label.value}" class="cmp-adaptiveform-button__text">${button.label.value}</span>
+    <span data-sly-test="${button.label.value}" class="cmp-adaptiveform-button__text cmp-adaptiveform-button__base_text">${button.label.value}</span>
```

Comment:
Why do you need this change ? There is already a class called `cmp-adaptiveform-button__text`

Reply:
it is needed because on the collateral page we have reset button and submit button as well. both of them have cmp-adaptiveform-button__text because they are inheriting the button component

Reply:
in that case, they would have other classes, you could use css selector with parent explicitly mentioned